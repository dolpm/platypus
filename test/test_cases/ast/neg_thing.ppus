thing MyThing = {
  im_a_string,
  im_an_int
}