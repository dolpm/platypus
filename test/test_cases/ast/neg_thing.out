Fatal error: exception Stdlib.Parsing.Parse_error
